product_cost_rollup_to_bom/views/product_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+<odoo>
+    <!-- product product form view -->
+    <record id="view_std_price_product_product" model="ir.ui.view">
+        <field name="name">view.std.price.product.product</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="before">
+                <field name="std_cost_update_date" readonly="1" />
```

Comment:
Also needed: the Product from `<button name="button_bom_cost" string="Compute Price from BoM" ...>` should be visible also for standard price products.

Reply:
It will be visible for standard price costing method products.
`attrs="{'invisible': ['|', ('bom_count', '=', 0), '&amp;', ('valuation', '=', 'real_time'), ('cost_method', '=', 'fifo')]}"`
attrs given in the base.